### ISSUE TITLE
[UI] Go to Line does not work when line number is 1

### ISSUE THREAD

**Issue by arku** [taskcluster/taskcluster#985]
### Steps to reproduce

   1. Navigate to http://localhost:5080/tasks/fRav6OG-Sbij116H7gfqag/runs/0/logs/https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask%2FfRav6OG-Sbij116H7gfqag%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log
   2. Click the 'Go to Line' icon

![image](https://user-images.githubusercontent.com/7039523/60820695-99781d00-a167-11e9-8906-de90a477cc4b.png)

   3. Enter `1` in the dialog that shows up and then click on the "Go To Line" button

**Expected:** Log jumps to the the first line.

**Actual:** Log does not jump to the first line.

**Comment by ss77995ss** [comment]
Hi, I just found that when we give `scrollToLine` value to `1`, scrollToIndex would be `-1` but actually number should be `0`. Maybe it's a bug from `react-lazylog`?

<img width="1279" alt="Screen Shot 2019-07-19 at 00 39 46" src="https://user-images.githubusercontent.com/28031139/61475692-f2dd1900-a9bd-11e9-95c9-e94f6cfe571b.png">

**Comment by helfi92** [comment]
Yeah, I think the bug is in `react-lazylog`.

**Comment by arku** [comment]
@helfi92 This bug was fixed in https://github.com/mozilla-frontend-infra/react-lazylog/pull/43

Fun fact, the above-mentioned PR wasn't intended to fix this issue :smile: I updated the description to help others understand the issue. Let me know if it is unclear.

I have submitted a PR upgrading react-lazylog to 4.2.0 [here](https://github.com/taskcluster/taskcluster/pull/1070) with which we can close this issue.

Check it when you get a chance.
